lib/esm-utils.js

```diff
@@ -0,0 +1,27 @@
+// This file is allowed to use async/await because it is not exposed to browsers (see the `eslintrc`),
```

Comment:
This file should be set into our "bin/cli" folder?

Reply:
If we decide to move our lib/cli folder to ES2017/8, then yes, that's a good idea, although it's a bit weird having it there because it has nothing to do with CLI.

Reply:
I decided not to move it to `lib/cli` because it has nothing to do with `cli`, and is needed from `mocha.js` (conditionally), so it doesn't make sense to have a dependency _from_ `mocha.js` to `cli`.

Of course, if you disagree, I will promptly move it there.